_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_hw_par.cpp

```diff
@@ -2420,7 +2420,7 @@ mfxStatus CheckVideoParam(MfxVideoParam& par, ENCODE_CAPS_HEVC const & caps, boo
         );
         //On Gen10+ VDEnc SAO for only Luma/Chroma in CQP mode isn't supported by driver until real customer usage
         //For TU 1 and TU 4 SAO isn't supported due to HuC restrictions, for TU 7 SAO isn't supported at all
-        if (par.m_platform >= MFX_HW_CNL &&
+        if ((par.m_platform >= MFX_HW_CNL && par.m_platform < MFX_HW_ICL) &&
```

Comment:
Hm. Isn't that just `par.m_platform == MFX_HW_CNL`?

Reply:
Yes, of course.